iXBRLViewerPlugin/iXBRLViewer.py

```diff
@@ -323,6 +342,27 @@ def addFact(self, f):
         self.taxonomyData["facts"][f.id] = factData
         self.addConcept(f.concept)
 
+    def measureString(self, measure, readable):
+        if readable:
+            return measure.localName.title() if measure.localName.islower() else measure.localName
+        else:
+            return self.nsmap.qname(measure)
+
+    def unitString(self, unit, readable):
```

Comment:
Curious why we have to turn this into a string instead of leaving it as a list, I would assume that would be preferred?

Reply:
I had in my head that the JSON was pseudo-OIM.  `"u":[[numers...],[denoms...]]` (e.g., `"u":[["iso4217:USD"]]`).

@paulwarren-wk Do you have thoughts on the design here?

Reply:
My intent was that additional information for the unit (a readable label, a monetary flag) be stored in a map somewhere, so having it in string representation aids in using it as a key to said map. I just completed a rework and I've included references to the OIM format in relevant docstrings.